package.json

```diff
@@ -23,7 +23,7 @@
     }
   },
   "author": "",
-  "devDependencies": {
+  "dependencies": {
     "babel-core": "^6.26.3",
     "babel-jest": "^24.1.0",
     "babel-preset-env": "^1.7.0",
```

Comment:
I think that since we're using webpack to build the viewer, there's no difference between dependencies and devDependencies.  webpack just pulls in what's needed when it creates the bundle.  It does make a difference for plugins: making them dependencies means that when a plugin lists this project as a dependency (or devDependency), all of this project's dependencies get pulled in as well, which is necessary as the plugin project builds a complete viewer.

Without this, the plugin would have to repeat all of this project's devDependencies which would be a maintenance pain.

Reply:
Ok, I trust you, but since I'm not familiar with webpack, could you give more explanation?  My high level intuition was:
1. For devDependencies, download those tools locally to assist with development.
2. For dependencies, when building for production, concatenate all the dependencies (not devDependencies) into a single JS file and minify it.

Based on your reply, I'm probably missing some extra step that filters out and only includes certain dependencies into the minified output file, but what is that step?

Reply:
I'm not following the intent here. Is the idea that new plugins should reuse the webpack config from a dependency (ixbrl-viewer)?

@brettkail-wk webpack doesn't install dependencies itself. npm, yarn, etc. handles that. Webpack doesn't know or care how a dependency was installed. It figures out what to bundle independently of these settings